test_data/asm/bit_access.asm

```diff
@@ -15,6 +15,7 @@ pil{
 // Wraps a value in Y to 32 bits.
 // Requires 0 <= Y < 2**33
 instr wrap Y -> X { Y = X + wrap_bit * 2**32, X = XB1 + 0x100 * XB2 + 0x10000 * XB3 + 0x1000000 * XB4 }
+instr jmp l: label { pc' = l }
```

Comment:
Could be
```suggestion
instr loop { pc' = pc }
```

Reply:
I used `jmp` when it already existed and introduced `loop` elsewhere